mne_bids/commands/mne_bids_raw_to_bids.py

```diff
@@ -54,14 +54,18 @@ def run():
     parser.add_option('--overwrite', dest='overwrite',
                       help=("Boolean. Whether to overwrite existing data"),
                       metavar='ow')
+    parser.add_option('--allow_maxshield', dest='allow_maxshield',
+                      help=("Boolean. Whether to allow non Maxfiltered data."),
+                      metavar='mxf')
```

Comment:
does this actually work? What command do you use? I think you may have to do `action="store_true"` or something to that effect

Reply:
Works like a charm.
`mne_bids raw_to_bids --subject_id sub01 --task rest --raw emo01.fif --output_path ./study --allow_maxshield True`
![image](https://user-images.githubusercontent.com/4716235/53400727-c8f36d80-39ae-11e9-8136-a0833fbf1000.png)

Reply:
The problem with this approach is that user needs to specify `True`. What if they provided `true` instead? The command line approach is to do:

mne_bids raw_to_bids --subject_id sub01 --task rest --raw emo01.fif --output_path ./study --allow_maxshield

It's similar to what you do with a parameter like verbose